assets/src/scss/layout/_navbar.scss

```diff
@@ -227,6 +226,7 @@ a.nav-link {
 
   @include medium-and-less {
     display: block;
+    right: 0;
```

Comment:
If you switch to `dir="rtl"` the close button is stuck on the wrong side, I think `insert-inline-end` would work better ?

Reply:
I figured out some minutes ago that the issue is bacause`insert-inline-end` is not supported to that iOS used version (12) (https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-start). Only it has a full supported from version `14.5`

I'm just working on that doing the same as you suggested !